modules/nw-multus-remove-pod.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+
+[id="nw-multus-remove-pod_{context}"]
+= Removing a Pod from an additional network
+
+You can remove a Pod from an additional network.
+
+.Prerequisites
+
+* Install the OpenShift Command-line Interface (CLI), commonly known as `oc`.
+* You must log in to the cluster.
+
+.Procedure
+
+To add a Pod to an additional network, complete the following steps:
+
+. Edit the Pod resource definition by running the following command. Replace
+`<name>` with the name of the Pod to edit.
++
+----
+$ oc edit pod <name>
+----
+
+. Update the `annotations` mapping to remove the additional network from the
+Pod:
+
+.. If you are removing all additional networks from a Pod, remove the
+`k8s.v1.cni.cncf.io/networks` parameter from the Pod resource definition as in
+the following example:
++
++
+[source,yaml] 
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: example-pod
+  annotations: {}
+spec:
+  containers:
+  - name: example-pod
+    command: ["/bin/bash", "-c", "sleep 2000000000000"]
+    image: centos/tools
+----
+
+.. If you are removing a specific additional network from a Pod, update the
+`k8s.v1.cni.cncf.io/networks` parameter by removing the name of the
+`AdditionalNetworkAttachment` for the additional network. In the following
```

Comment:
This isn't helpful. I'd remove the sentence that starts with "In the following" and add an annotation that line that says to ensure that only the networks that you want to keep remain in the annotation.